apps/ex_wire/lib/ex_wire/struct/block.ex

```diff
@@ -21,6 +21,16 @@ defmodule ExWire.Struct.Block do
           ommers: list(Header.t())
         }
 
+  @spec new(Blockchain.Block.t()) :: t()
```

Comment:
alias

Reply:
Is it weird to alias `Block` to mean a different `Block` than the containing module that is named `Block`? In other languages, I've used the full path for the module that isn't the current one to indicate that it's not what the reader might assume it is.

Reply:
**It really depends on what you agree to.** I think our problem is that we're using aliases somewhere and on others we don't. We've yet to define when to alias and when not to for the sake of consistency. A good thing about Erlang is that it doesn't have aliases :)

So if you decide to alias, there's a thing `alias as`, if not, then don't. We should then disable `AliasUsage` in Credo and come up with a definition on when a developer should alias and when not... even for the sake of public contributions and our code consistency.

Reply:
FYI, I only alias because Credo tells us to. I think it's easier to have full names everywhere since it makes it simple to look up a module (versus reading "Sync.. is that CLI.Sync or ExWire.Sync? Let's look at the aliases). However, our current credo settings tells us we must alias **every module**. That said, `Blockchain.Block` versus `Block.Header` is completely terrible and my fault. We originally had `Blockchain.Block.Header`, but I realized that `EVM` needed `Block.Header`, so it was moved out. @germsvel has discussed several times rolling back this change and possible merging the EVM and Blockchain apps.